src/pudl/extract/ferc1.py

```diff
@@ -174,6 +174,14 @@
         "dbf": "f1_dacs_epda",
         "xbrl": "summary_of_depreciation_and_amortization_charges_section_a_336",
     },
+    "accumulated_depreciation_plant_ferc1": {
```

Comment:
Not a huge fan of these table names, but also not sure what would be better... if you have suggestions!

Reply:
it could be `utility_plant_depreciation_ferc1`

Reply:
What would the other tables be in relation to that -- the one organized by functional classification, and the one containing the unstructured data?

Reply:
I went with:
* `electric_plant_depreciation_changes_ferc1` and
* `electric_plant_depreciation_functional_ferc1`

And I guess we could do an `electric_plant_depreciation_changes_unstructured_ferc1` or something.  We should come up with a convention for the unstructured table names.